src/components/html-injection/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const widgetName = "html-injection";
+export const widgetDisplayName = "Custom HTML code";
+export const widgetCategory = "Advanced";
+export const widgetSelector = "widget";
```

Comment:
`export const widgetSelector = "widget";` - this selector (and few others here) is generic (taken from tutorial) and may conflict with other widgets, if not renamed.